UnityProject/Assets/ScriptableObjects/Traits/CommonTraitsSingleton.asset

```diff
@@ -63,6 +63,7 @@ MonoBehaviour:
   Loomable: {fileID: 11400000, guid: 239e6bd28c3fc4442bf2dc5420eada94, type: 2}
   CanisterFillable: {fileID: 11400000, guid: a88f1555ca8abf643aea180c637f5329, type: 2}
   Breakable: {fileID: 11400000, guid: 86782de91b339c1438d9f4e310d321f4, type: 2}
+  LightableSurface: {fileID: 11400000, guid: 7410b4f6e26235e4f93aa1ebcbab0cec, type: 2}
```

Comment:
I would say to serialize reference to this trait on the match script as it probably won't be used much elsewhere. There's a few other traits that probably don't belong here too (though that isn't your problem : ) ).